server/helpers/index.js

```diff
@@ -1,6 +1,189 @@
+import https from 'https'
+
+import Katathon from '../models/katathonModel'
+
 export const getNextEvent = events => {
   const newEvents = events.filter(event => event.date >= Date.now().valueOf())
   return newEvents.concat().sort((a, b) => a.date < b.date ? -1 : 1)[0]
 }
 
 export const dateToTimestamp = date => new Date(date).valueOf()
+
+const getHistoricCompletedKatas = (userName, totalPages) => {
+  // This will be a list of all ongoing requests
+  const allResults = []
+  let historicCompleted = []
+  for (let i = 1; i < totalPages; i++) {
+    // Request is wrapped in a promise so that Promise.all can run when all requests
+    // have resolved
+    const result = new Promise((resolve, reject) => {
+      https.get(
+        `https://www.codewars.com/api/v1/users/${userName}/code-challenges/completed?page=${i}`,
```

Comment:
This doesn't really qualify as a helper when it's nearly 60 lines long and makes an API call. Try to break it down and compose it of smaller helpers that are pure functions. The below inline function could be a good example to start with.